docs/source/histogram.rst

```diff
@@ -0,0 +1,84 @@
+.. Copyright (c) 2016, Johan Mabille, Sylvain Corlay and Wolf Vollprecht
+
+   Distributed under the terms of the BSD 3-Clause License.
+
+   The full license is in the file LICENSE, distributed with this software.
+
+Histogram
+=========
+
+Basic usage
+-----------
+
+.. note::
+
+    .. code-block::
+
+        xt::histogram(a, bins[, weights][,density])
+        xt::histogram_bin_edges(a, bins[, weights][, left, right][, bins][, mode])
+
+    Any of the options ``[...]`` can be omitted (though the order must be preserved). The defaults are:
+
+    *   ``weights = xt::ones(data.shape())``
+    *   ``density = false``
+    *   ``left = xt::amin(data)[0]``
```

Comment:
I think we shouldn't advocate the use of the square brackets operator as it's implementaiton is not really consistent and might be removed in the future. I think it might be better to use `(0)` here.

Reply:
@wolfv Fair enough.

Reply:
Or do you feel that they can stay in the code?